packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -183,6 +184,8 @@ const DEFAULT_AGGREGATION_SETTINGS = Object.freeze({
   showOnTop: false,
 });
 
+const UNFORMATTED_DATE_PATTERN = 'yyyy-MM-dd HH:mm:ss.SSSSSSSSS z';
```

Comment:
I think the intent is the value would then be immediately usable as part of a query, in which case you'll want a T separator to be included.